index.js

```diff
@@ -68,7 +69,33 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header");
+header.addEventListener("click", function () { toggleColor(header) });
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector(".like-button");
+const likeParagraph = document.querySelector(".likes");
+
+likeButton.addEventListener("click", function(){
+  const likes = parseInt(likeParagraph.textContent);
+  const addLikes = likes + 1;
+  likeParagraph.textContent = `${addLikes} likes`; 
+});
```

Comment:
Looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. Nice work!